python/paddle/fluid/framework.py

```diff
@@ -749,16 +954,7 @@ def detach(self):
                     y = x.detach()
 
         """
-        if in_dygraph_mode():
-            new_var = self._cloneVar()
-            self.block.append_op(
-                type="assign",
-                inputs={'X': [self]},
-                outputs={'Out': [new_var]},
-                stop_gradient=True)
-            return new_var
-        else:
-            raise AttributeError("static graph model DO NOT supprt detach")
+        pass
```

Comment:
Raise error or just remove this method?

Reply:
Thanks a lot. For generating English docs, this APIs are kept currently, which should be refined together with upgrading APIs.